partner_firstname/readme/CONTRIBUTORS.rst

```diff
@@ -18,3 +18,6 @@
 * Graeme Gellatly <graeme@o4sb.com>
 * Laurent Mignon <laurent.mignon@acsone.eu>
 * Bjorn Billen <bjorn.billen@dynapps.be>
+* `Aion Tech <https://aiontech.company/>`_:
```

Comment:
Let's leave these automatic changes to the bot, so the commit contains only the actual changes.

Reply:
This is not an automatic change, this is me adding myself as a contributor.